xicam/plugins/__init__.py

```diff
@@ -40,7 +41,8 @@
 
 from yapsy.PluginManager import NormalizePluginNameForModuleName, imp, log
 import importlib.util
-from collections import deque
+from queue import LifoQueue
```

Comment:
Why do we want a stack instead of a `Queue` here?

Reply:
after talking to @ronpandolfi, it needs to be a stack to handle plugin dependencies properly:
a -> b -> c, put(b), put(c)